DQM/SiStripMonitorCluster/python/SiStripMonitorCluster_cfi.py

```diff
@@ -249,6 +291,26 @@
         xmin = cms.double(-0.5)
         ),
 
+     TProfNClusStrip = cms.PSet(
+        Nbinsx = cms.int32(3565),
+        xmin = cms.double(0),
+        xmax = cms.double(3564),
+        Nbinsy = cms.int32(200),
+        ymin = cms.double(-0.5),
+        ymax = cms.double(99999.5),
+        globalswitchon = cms.bool(True)
+        ),
+
+     TProfNClusPixel = cms.PSet(
+        Nbinsx = cms.int32(3565),
+        xmin = cms.double(0),
+        xmax = cms.double(3564),
+        Nbinsy = cms.int32(200),
```

Comment:
How many of instances of these will appear?
3565x200 ~1 M bins is already fairly large by itself.
Will this run in PromptReco?